### DISCUSSION TITLE
Beta feature: Added keywords to filter pizza Foodish images in v1.2.0

### DISCUSSION THREAD

**Discussion by surhud004** [surhud004/Foodish#14] [Announcements]
#### `BETA FEATURE`:

Rolling out today as a beta feature, now you can filter the `pizza` category Foodish images and here's how you can achieve it:

* Refer to the `/images/pizza/` endpoint on the UI or via the API.
* Add a `?` followed by `keyword=` followed by one of the keywords listed below.
* You should be able to filter specific pizza images based on the keyword.

* An example use-case to filter only `vegetarian` pizzas could be as follows:
```
https://foodish-api.herokuapp.com/images/pizza/?keyword=vegetarian
```
or if using the `/api` endpoint:
```
https://foodish-api.herokuapp.com/api/images/pizza/?keyword=vegetarian
```

* While using this feature, if a new random pizza image is not being fetched, it probably means there is only one image associated with that keyword. You might want to try another keyword from the list given below.

* This feature is only available on the `/images/:food` endpoint. Make sure you are referring to the correct URL path (please refer to the [README](https://github.com/surhud004/Foodish#endpoints)).

* If a provided `keyword` does not match with any image, a random image will be fetched and returned as a response.

* Please [create a new issue](https://github.com/surhud004/Foodish/issues/new) for new keyword requests.

#### List of popular available `keyword` values:
* `vegetarian`
* `pepperoni`
* `margherita`
* `pineapple`
* `mushroom`
* `olives`
* `cheese`

##### Note: Since this is a beta feature, the README docs won't be updated.

**Comment by thushanr32** [comment]
Hi,
The Heroku app is down, will it be coming up anytime soon?

Thanks

**Reply by surhud004** [reply]
Hello @thushanr5, a new version of the Foodish API v2 has been released! Read more: https://github.com/surhud004/Foodish/discussions/19
